kodilla-stream/src/test/java/com/kodilla/stream/world/WorldTestSuite.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.stream.world;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WorldTestSuite {
+    @Test
+    public void testGetPeopleQuantity(){
+        //Given
+        BigDecimal expected = new BigDecimal("50");
+
+        List<Continent>continentList = new ArrayList<>();
+        continentList.add(new Continent(countryListGenerator(5)));
+        continentList.add(new Continent(countryListGenerator(10)));
+        continentList.add(new Continent(countryListGenerator(20)));
+        continentList.add(new Continent(countryListGenerator(15)));
+
+        World world = new World(continentList);
+        //When
+        BigDecimal result = world.getPeopleQuantity();
+        //Then
+        Assert.assertEquals(expected, result);
+    }
+
+    private List<Country> countryListGenerator(int size){
+        List<Country> countries = new ArrayList<>();
+        BigDecimal bigDecimal = BigDecimal.ZERO;
+        for (int i = 0; i <size ; i++) {
+            countries.add(new Country(bigDecimal.add(BigDecimal.ONE)));
+        }
+        return countries;
```

Comment:
czy na pewno jest senso zeby dodawac kraje z populacja 1, 2, 3 ? Wydaje mi sie, ze to tylko zaciemnia kod. Mozna to zastapic zahardcodowanymi wartosciami. Ewentualnie uzyc ponownie parameters-test